### ISSUE TITLE
Can't subscribe to newsletter

### ISSUE THREAD

**Issue by dferber90** [brianlovin/briOS#1753]
I tried to subscribe to Overthought both from your unbelievably good new site (using the Subscribe button at the top of the articles list) and directly from your Twitter profile and I get this error message preventing me from confirming my subscription when I click the link sent in Revues confirmation email (in both cases):

![image](https://user-images.githubusercontent.com/1765075/139739180-9753de46-2c8d-4480-a61c-ec44775313f6.png)

**Comment by brianlovin** [comment]
Oof, thanks for the heads up — it might be that you're already subscribed? If you wanna DM me your email or share it in a comment here, I can double check.

**Comment by dferber90** [comment]
True, I used to be subscribed through Mailchimp but had unsubscribed at one point iirc. 

I checked my email history and the last mail I received from you is titled "How to give a great product design portfolio presentation" from the 7th of May. Since you've published new writing since, it seemed like I was unsubscribed. I can't say for sure if I was or was not subscribed before.

Anyways, I used ye olde [task specific gmail address (+)](https://support.google.com/a/users/answer/9308648?hl=en) trick to subscribe with a different address and it worked. So this seems to be something that's just related to my specific account and not a general problem. I'll close this, sorry for the sorta false alarm!
